src/pages/Home.tsx

```diff
@@ -1,40 +1,63 @@
 import React, { useState, useEffect } from "react";
 import axios from "axios";
-import { Pagination, Paper, Toolbar, Typography, AppBar } from "@mui/material";
+import { Pagination, Paper } from "@mui/material";
 import NFTCard from "../components/card/NFTCard";
+import { StyledCardContainer } from "./styledPages/StyledCardContainer.styled";
+import { NFTGrid } from "../components/card/styledCard/NFTGrid.styled";
 
 const NFTsPerPage = 5;
 
-const Home = () => {
-  const [nfts, setNFTs] = useState<any[]>([]);
+interface NFT {
+  id: string;
+  name: string;
+  description: string;
+  image_url: string;
+  background_color?: string;
+  asset_contract: { chain_identifier: string };
+}
+
+const Home: React.FC = () => {
+  const [nfts, setNFTs] = useState<NFT[]>([]);
   const [loading, setLoading] = useState<boolean>(true);
   const [error, setError] = useState<string | null>(null);
   const [page, setPage] = useState(1);
 
-  const API_KEY = "5402e464f97c44968342dd3057b4a262";
-
-  const fetchNFTs = async () => {
+  const fetchNFTs = async (pageNumber: number) => {
     try {
       const response = await axios.get("https://api.opensea.io/api/v1/assets", {
         params: {
           order_direction: "asc",
-          offset: (page - 1) * NFTsPerPage,
-          limit: NFTsPerPage,
+          offset: (pageNumber - 1) * NFTsPerPage,
+          limit: NFTsPerPage + 5,
         },
         headers: {
-          "X-API-KEY": API_KEY,
+          "X-API-KEY": `${process.env.REACT_APP_API_KEY}`,
         },
       });
-      setNFTs(response.data.assets);
+
+      const allData: NFT[] = response.data.assets.map((asset: any) => ({
+        id: asset.id,
+        name: asset.name || "No Name",
+        description: asset.description || "No Description",
+        image_url: asset.image_url || "No Image URL",
+        background_color: asset.background_color || undefined,
+        asset_contract: asset.asset_contract || undefined,
+      }));
+
+      const filteredData: NFT[] = allData
+        .filter((nft: NFT) => nft.name !== "Ledger Legend Card")
```

Comment:
remove the filter and read opensea doc and how to fetch data

Reply:
Filter was to block NFT's which doesnt have any informations except image, i  change URL for fetching NFTs and this line of code is not needed anymore